### PR TITLE
[Feature-#5258][Bin] Add .bat script for all shell script.

### PR DESCRIPTION
**Pull request by FlechazoW**
## Purpose of this pull request

Add .bat script for all shell script.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: config/seatunnel-env.cmd [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+REM    http://www.apache.org/licenses/LICENSE-2.0
+REM
+REM Unless required by applicable law or agreed to in writing, software
+REM distributed under the License is distributed on an "AS IS" BASIS,
+REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+REM See the License for the specific language governing permissions and
+REM limitations under the License.
+
+REM Home directory of spark distribution.
+set "SPARK_HOME=C:\Program Files\spark"
```
Comment (lvyanquan) [inline config/seatunnel-env.cmd]: if SPARK_HOME is already defined, we should keep it as before.

File: seatunnel-core/seatunnel-starter/src/main/bin/stop-seatunnel-cluster.cmd [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+rem Unless required by applicable law or agreed to in writing, software
+rem distributed under the License is distributed on an "AS IS" BASIS,
+rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+rem See the License for the specific language governing permissions and
+rem limitations under the License.
+
+setlocal enabledelayedexpansion
+
+set "SEATUNNEL_DEFAULT_CLUSTER_NAME=seatunnel_default_cluster"
+set "SHOW_USAGE=Usage: stop-seatunnel-cluster.bat [options]"
```
Comment (lvyanquan) [inline seatunnel-core/seatunnel-starter/src/main/bin/stop-seatunnel-cluster.cmd]: "Options:
       -cn, --cluster      The name of the cluster to shut down (default: $SEATUNNEL_DEFAULT_CLUSTER_NAME) 
       -h, --help          Show the usage message"
 
  -cn, --cluster should be displayed.

**Comment by FlechazoW** [comment]
@lvyanquan @EricJoy2048  PTAL

**Comment by lvyanquan** [comment]
Overall, looks good to me.

**Review by lvyanquan — APPROVED** [review]

**Review by davidzollo — APPROVED** [review]
good job

**Review by EricJoy2048 — APPROVED** [review]

**Pull request merged**